src/cocaine/burlak/metrics/procfs.py

```diff
@@ -381,6 +385,11 @@ def read(self):
             drx = int(NET_TICKS_PER_SEC * (net2.rx - net1.rx))
             dtx = int(NET_TICKS_PER_SEC * (net2.tx - net1.tx))
 
+            rates = self.rates(iface)
```

Comment:
I can't find where `self._reates` is updated. It shouldn't be or should?

Reply:
Yep, it should be, fixed.